src/components/activity/screens/ActivityAdd/containers/ActivityAddContainer.js

```diff
@@ -22,33 +23,39 @@ class ActivityAddContainer extends React.Component {
         date: moment(),
         type: ActivityTypeEnum.CONTENT,
         contentSource: "",
-        meetingWho: "",
         tags: [],
         key: this.props.lastID + 1,
+        interlocutorKey: 0
```

Comment:
If what you wanted to do was linking a interlocutor when creating the activity, I think it is better to initialize it to `null`  to make sure the activity won't be created with the `0` id.